src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,8 @@
 // Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+export const UserInfo = ({ user }) => (
+  <>
+    <a className="UserInfo" href={user.email}>
```

Comment:
The href for the anchor tag is set to user.email. If it's meant to be a mailto link, it should be 'mailto:' + user.email